<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Send_Message">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>220</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Message to Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Send_File">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>491</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Send File to Server</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Message">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Inbox</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Send a Message</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_Inbox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>331</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
